### PR TITLE
Add Visual Studio Code support

### PR DESCRIPTION
**Pull request by luisdemanuel**

### CODE REVIEW

File: mackup/applications/vscode.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = vscode
```
Comment (lra) [inline mackup/applications/vscode.cfg]: The name should be the full real name, it's for display purpose

File: mackup/applications/vscode.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+[configuration_files]
+Library/Application Support/Code/User
```
Comment (lra) [inline mackup/applications/vscode.cfg]: nothing big in this thing?

**Comment by lra** [comment]
Can you add a line in the changelog? https://github.com/lra/mackup/blob/master/CONTRIBUTING.md

File: mackup/applications/vscode.cfg (hunk 2 above)
Comment (luisdemanuel) [inline mackup/applications/vscode.cfg]: No, only a keybindings and a settings JSON files

**Pull request merged**